src/pretix/presale/templates/pretixpresale/event/order_modify.html

```diff
@@ -61,7 +61,9 @@ <h4 class="panel-title">
                             {% for form in forms %}
                                 {% if form.pos.item != pos.item %}
                                     {# Add-Ons #}
-                                    <legend>+ {{ form.pos.item.name }}</legend>
+                                    <legend>+ {{ form.pos.item.name }}{% if form.pos.variation %}
+                                    – {{ form.pos.variation }}
```

Comment:
```suggestion
                                    – {{ form.pos.variation.value }}
```
Is a little more safe for the future, but otherwise this is great